draft-irtf-cfrg-concrete-hybrid-kems.md

```diff
@@ -155,7 +155,7 @@ define how they meet the Nominal Group interface described in
 {{HYBRID-KEMS}}.
 
 Group elements are elliptic curve points, represented as byte strings in the
-compressed representation defined by the Elliptic-Curve-Point-to-Octet-String
+uncompressed representation defined by the Elliptic-Curve-Point-to-Octet-String
```

Comment:
`Nelem` below needs an update. Perhaps also add parenthetical that this includes the type byte.